code/game/objects/items.dm

```diff
@@ -119,6 +119,7 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	var/list/sprite_sheets = null
 	var/list/sprite_sheets_inhand = null //Used to override inhand items. Use a single .dmi and suffix the icon states inside with _l and _r for each hand.
 	var/icon_override = null  //Used to override hardcoded clothing dmis in human clothing proc.
+	var/prioritize_species_over_icon_overide = FALSE //Do we want to overide the icon_overide and use species instead?
```

Comment:
Hesitant about adding a new item var and would prefer if only `icon_override` was used somehow. We seem to have a proc that might help: `/obj/item/clothing/proc/refit_for_species(target_species)` (though it's unused).